File: TTT VSC WPF/TicTacWPF/model/Spieler.cs
using System;

namespace model
{
    public class Spieler : ISpieler
    {
        private Farbe Farbe;
        private int SpielerID;
        private String Spielername = "";
        private int Wins;
        public Farbe getSpielerfarbe()
        {
            return this.Farbe;
        }

        public int getSpielerID()
        {
            return this.SpielerID;
        }

        public string getSpielerName()
        {
            return this.Spielername;
        }

        public void setSpielerName(string spielerName)
        {
            this.Spielername = spielerName.Trim().Substring(0, Math.Min(12, spielerName.Trim().Length));
        }

        public void gewonnen()
        {
            Wins++;
        }

        public int gewonneneSpiele()
        {
            return Wins;
        }

        public Spieler(string Name, Farbe Farbe, int ID)
        {
            this.setSpielerName(Name);
            this.Farbe = Farbe;
            this.SpielerID = ID;
        }
    }
}

File: TTT Python PySimpleGui/model/Spieler.py
from model.Farbe import Farbe


class Spieler(object):
    def __init__(self, Farbe: Farbe, Name: str = "Alice Little", Id: int = -1):
        self._name = Name
        self._id = Id
        self._wins = 0
        self._Farbe = Farbe

    @property
    def name(self) -> str:
        return self._name

    @property
    def id(self) -> int:
        return self._id

    @property
    def wins(self) -> int:
        return self._wins

    def inc_wins(self) -> None:
        self._wins += 1

    @property
    def Farbe(self) -> Farbe:
        return self._Farbe

    @Farbe.setter
    def Farbe(self, Farbe: Farbe):
        self._Farbe = Farbe
